develop.sh

```diff
@@ -3,4 +3,8 @@
 # This runs the development server. Use ./init.sh beforehand to install the
 # dependencies to node_modules/.
 
-docker run -it -w /bedrock -v $(pwd):/bedrock -p 3000:3000 node:14.15.0-buster npm start
+if [[ $OSTYPE == "msys" ]]; then
+    MSYS_NO_PATHCONV=1 docker run -it -w /bedrock -v $(pwd):/bedrock -p 3000:3000 node:14.15.0-buster npm start
```

Comment:
Since the lines seem the same, maybe you could simply just have something like

```
if [[ $OSTYPE == "msys" ]]; then
  export MSYS_NO_PATHCONV=1
fi
```

before the command ?

Reply:
Do we want to disable path conversion at system level? Or do I maybe misunderstand the scope of the export command?

Reply:
The scope is until the end of the script (and it will not affect the bash instance from which the script was run).